masterdata/configuration/concepts/programAttributes.csv

```diff
@@ -0,0 +1,16 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Short name:en,Description:en,Data class,Data type,Answers,Members,_version:1,_order:3000
+0c9c09cf-39c8-4ce7-a656-5d8c7e2c57e1,,,Initial Stage,Initial Stage,,Misc,N/A,,,,
```

Comment:
@deepthi-mantena I see, you have used v1 UUID, this could easily lead to duplicates, recommended to use UUID v4.

Reply:
@mohan-13 The UUID 0c9c09cf-39c8-4ce7-a656-5d8c7e2c57e1 is a v4  UUID.

In UUIDs, the version is specified by the first digit of the third group. In this case, 4ce7 indicates that it is a UUID v4, as the first digit is 4.And this has been randomly generated by Version 4 UUID Generator.

Reply:
Also removed the sequence of v4 UUId's as discussed and generated random for every entry.